stable/superset/values.yaml

```diff
@@ -47,6 +47,14 @@ configFile: |-
   # Set this API key to enable Mapbox visualizations
   MAPBOX_API_KEY = ''
 
+## Extra environment variables that will be passed onto deployment pod
+##
+env: {}
```

Comment:
I think that the "best practice" name for this variable would be `extraEnv` (and `extraEnvFromSecret`). My understanding is that the `charts` repo is trying to use `extraEnv` going forward.

cc @timstoop who first brought this to my attention :)